x-pack/plugin/core/src/main/config/log4j2.properties

```diff
@@ -104,6 +104,21 @@ appender.audit_rolling.policies.size.size = 1GB
 appender.audit_rolling.strategy.type = DefaultRolloverStrategy
 appender.audit_rolling.strategy.fileIndex = nomax
 
+appender.behavioral_analytics_rolling.type = RollingFile
```

Comment:
@rjernst could your team help reviewing this part? We had a discussion last month regarding the usage of log4j to log events for search analytics. This is the implementation so would love to get your feedback on the viability of the solution.

Reply:
This has been removed as we are now ingesting events directly.